<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSettings</class>
 <widget class="QDialog" name="dlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bbdigitalminutes.qrc">
    <normaloff>:/resources/settings.png</normaloff>:/resources/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="gbApp">
     <property name="title">
      <string> Application settings </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lblAppLanguage">
          <property name="text">
           <string>Application language: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbLang">
          <property name="toolTip">
           <string>Language of the application GUI</string>
          </property>
          <item>
           <property name="text">
            <string>en (English)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>hu (Magyar)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRetranslate">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="bbdigitalminutes.qrc">
            <normaloff>:/resources/reset.png</normaloff>:/resources/reset.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>63</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkReloadSizePos">
        <property name="text">
         <string>Reload size and position of main window from the last run.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkReloadLastMinute">
        <property name="text">
         <string>On start automatically reload last minute if it is not closed.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="chkPlaySounds">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Play sounds</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbEditSound">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="bbdigitalminutes.qrc">
            <normaloff>:/resources/edit.png</normaloff>:/resources/edit.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPlay">
     <property name="title">
      <string> Play settings </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblCountQuarter">
          <property name="text">
           <string>Number of play periods:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledCountQuarter">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblLengthPeriod">
          <property name="text">
           <string>Length of play periods (min.):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledLengthPeriod">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="chkAllowExtendedTime">
          <property name="text">
           <string>Allow overtime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblLengthOvertime">
          <property name="text">
           <string>Length of overtime (min.):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledLengthOvertime">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblLengthOffense">
          <property name="text">
           <string>Length of offense play (min.):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledLengthOffense">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkOffensTimeUsed">
        <property name="text">
         <string>Offense time is in use during game</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblLengthOffenseExtended">
          <property name="text">
           <string>Length of offense play extension (min.):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledLengthOffenseExtended">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="lblLengthTimeout">
          <property name="text">
           <string>Length of timeout (min.):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledLengthTimeout">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbReset">
       <property name="text">
        <string>Reset to default</string>
       </property>
       <property name="icon">
        <iconset resource="bbdigitalminutes.qrc">
         <normaloff>:/resources/reset.png</normaloff>:/resources/reset.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="bbdigitalminutes.qrc">
         <normaloff>:/resources/ok.png</normaloff>:/resources/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="bbdigitalminutes.qrc">
         <normaloff>:/resources/cancel.png</normaloff>:/resources/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbLang</tabstop>
  <tabstop>pbRetranslate</tabstop>
  <tabstop>chkReloadSizePos</tabstop>
  <tabstop>chkReloadLastMinute</tabstop>
  <tabstop>chkPlaySounds</tabstop>
  <tabstop>pbEditSound</tabstop>
  <tabstop>ledCountQuarter</tabstop>
  <tabstop>ledLengthPeriod</tabstop>
  <tabstop>chkAllowExtendedTime</tabstop>
  <tabstop>ledLengthOvertime</tabstop>
  <tabstop>ledLengthOffense</tabstop>
  <tabstop>ledLengthOffenseExtended</tabstop>
  <tabstop>ledLengthTimeout</tabstop>
  <tabstop>pbReset</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="bbdigitalminutes.qrc"/>
 </resources>
 <connections/>
</ui>
